src/cargo/util/command_prelude.rs

```diff
@@ -66,10 +68,13 @@ pub trait CommandExt: Sized {
 
     fn arg_jobs(self) -> Self {
         self._arg(
-            opt("jobs", "Number of parallel jobs, defaults to # of CPUs")
-                .short('j')
-                .value_name("N")
-                .allow_hyphen_values(true),
+            opt(
+                "jobs",
+                "Number or \"default\" of parallel jobs, defaults to # of CPUs",
```

Comment:
Mine below is clearer but way too long 😢. I don't have a good solution here.

Or maybe, it needn't be mentioned here, as it is more like an “advanced” feature?

```suggestion
                "Number of parallel jobs, defaults to # of CPUs. Set \"default\" to unset to defaults",
```

Reply:
Do we like new lines? If yes, it can become:
```
  -j, --jobs <N>                Number of parallel jobs, defaults to # of CPUs.Set "default"
                                to unset to defaults
      --keep-going              Do not abort the build as soon as there is an error (unstable)
```

Reply:
We've enabled `wrap_help` in clap since <https://github.com/rust-lang/cargo/pull/12013>. It would be a bit hard to read on small screens.

```
  -j, --jobs <N>
          Number of parallel jobs, defaults to # of CPUs.
          Set "default"
          to unset to defaults
```

As mentioned, I feel like it is an advanced feature and may not need to mention here. I have no strong preference though. Can merge the version without a newline.

Reply:
Reverted it altogether, IMO you're right that it feels that is is an advanced feature.